modules/class-simplechart-mexp.php

```diff
@@ -3,8 +3,10 @@
 function simplechart_mexp_init( $services ){
 
 	//should we remove all other Media Explorer services?
-	global $simplechart;
-	if ( true === $simplechart->get_config( 'clear_mexp_default_svcs' ) ){
+	$clear_defaults = ! defined( 'WPCOM_IS_VIP_ENV' ) || ! WPCOM_IS_VIP_ENV;
```

Comment:
I think is correct as is, but noting that this will return false on dev envs even if they're targeting the VIP environment. (I still think it's correct since you could change that using the filter)